tinygrad/ops.py

```diff
@@ -114,7 +114,8 @@ def consume_flops(self):
 from tinygrad.shape.shapetracker import ShapeTracker
 shape_fxn_for_op: Dict[Op, Callable] = {
   UnaryOps.CAST: lambda self,dtype: (self.shape, dtype, self.consume_flops()),   # cast uses no flops
-  **{op:lambda self: (self.shape, self.dtype, self.consume_flops() + prod(self.shape)) for op in UnaryOps if op != UnaryOps.CAST},
+  UnaryOps.BITCAST: lambda self,new_dtype: (((prod(self.shape) * self.dtype.itemsize * 8) // (new_dtype.itemsize * 8),), new_dtype, self.consume_flops()),  # bitcast may change shape
```

Comment:
I don't think bitcast should be allowed to change shape, it doesn't make sense.

Reply:
Yeah I was not sure which way to go, so chose to maintain parity with TensorFlow's bitcast which allows to change shape.